lib/github_backend.rb

```diff
@@ -232,8 +232,8 @@ def get_repos(opts)
 		if opts.repos != nil
 			repos = repos.concat(opts.repos)
 		end
-		if opts.orgas != nil
-			opts.orgas.each do |orga|
+		if opts.orgs != nil
+			opts.orgs.each do |org|
```

Comment:
Unused block argument - `org`. You can omit the argument if you don't care about it.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.